### PR TITLE
Makefile remove-whitespaces and wf change

### PR DESCRIPTION
**Pull request by ElaiShalevRH**
Added makefile target 'remove-whitespaces' which target application.properties files
and small change on the workflow to work only on PR's

### CODE REVIEW

**Comment by ElaiShalevRH** [comment]
@masayag @gabriel-farache

File: Makefile [hunk 1]
```diff
@@ -185,3 +185,8 @@ push-manifests: prepare-workdir
 	cd $(WORKDIR)
 	@$(CONTAINER_ENGINE) run --rm -v $(WORKDIR):/workdir -w /workdir \
 		$(LINUX_IMAGE) /bin/bash -c "${SCRIPTS_DIR}/push_manifests.sh '$(GIT_USER_NAME)' $(GIT_USER_EMAIL) $(GIT_TOKEN) $(PR_OR_COMMIT_URL) $(DEPLOYMENT_REPO) $(DEPLOYMENT_BRANCH) $(WORKFLOW_ID) $(APPLICATION_ID) $(IMAGE_NAME) $(IMAGE_TAG)"
+
+remove-whitespaces:
+	@echo "Removing all whitespaces from application.properties files"
+	@find . -type f -name "application.properties" -exec sed -i 's/[[:space:]]\+$$//' {} +
+	@echo "Whitespaces were removed."
```
Comment (gabriel-farache) [inline Makefile:192]: nice 
May be worthy to tell the user to execute this target if/when the check-withespaces CI fails

**Review by gabriel-farache — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -185,3 +185,8 @@ push-manifests: prepare-workdir
 	cd $(WORKDIR)
 	@$(CONTAINER_ENGINE) run --rm -v $(WORKDIR):/workdir -w /workdir \
 		$(LINUX_IMAGE) /bin/bash -c "${SCRIPTS_DIR}/push_manifests.sh '$(GIT_USER_NAME)' $(GIT_USER_EMAIL) $(GIT_TOKEN) $(PR_OR_COMMIT_URL) $(DEPLOYMENT_REPO) $(DEPLOYMENT_BRANCH) $(WORKFLOW_ID) $(APPLICATION_ID) $(IMAGE_NAME) $(IMAGE_TAG)"
+
+remove-whitespaces:
+	@echo "Removing all whitespaces from application.properties files"
```
Comment (masayag) [inline Makefile:190]: just to be on the safe-side, let's add replace `whitespaces` with `trailing whitespaces`

**Review by gabriel-farache — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (masayag) [inline Makefile:192]: in this message as well (replace whitespaces with trailing whitespaces)

**Review by masayag — APPROVED** [review]

**Pull request merged**